src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -186,19 +186,19 @@ public static void deleteColony(final int id)
                     colonyWorlds.add(world);
                 }
             }
-            Log.getLogger().info("Removing buildings for " + id);
-            for (final AbstractBuilding building : new ArrayList<>(colony.getBuildings().values()))
+            if(canDestroy)
             {
-                final BlockPos location = building.getLocation();
-                Log.getLogger().info("Delete Building at " + location);
-                building.deconstruct();
-                building.destroy();
-                for (final World world : colonyWorlds)
-                {
-                    if (world.getBlockState(location).getBlock() instanceof AbstractBlockHut)
-                    {
-                        Log.getLogger().info("Found Block, deleting " + world.getBlockState(location).getBlock());
-                        world.setBlockToAir(location);
+                Log.getLogger().info("Removing buildings for " + id);
+                for (final AbstractBuilding building : new ArrayList<>(colony.getBuildings().values())) {
+                    final BlockPos location = building.getLocation();
+                    Log.getLogger().info("Delete Building at " + location);
+                    building.deconstruct();
+                    building.destroy();
+                    for (final World world : colonyWorlds) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)